priv/gettext/en/LC_MESSAGES/apns.po

```diff
@@ -64,7 +64,7 @@ msgstr "Come in and call now 👉"
 #, elixir-format
 #: lib/t/push_notifications/apns.ex:109
 msgid "%{name} пригласил%{gender_a} тебя на дэйт!"
-msgstr "You've been asked out on a date by %{name}!"
+msgstr "%{name} invites you on a date!"
```

Comment:
when you edit push, you are supposed to change it in minimum like 4 places:
* dgettext (where the push is created)
* apns.po (master file of texts of all pushes)
* ru apns.pot
* en apns.pot

the way you should change pushes is to first change it just in code (dgettext), then run `mix gettext.extract --merge` , which will automatically trigger changes in all the other 3 places, and then if necessary correct the translations in relevant .pot files.